libs/application/templates/estate/src/forms/sharedSections/deceasedInfoFields.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  buildDescriptionField,
+  buildKeyValueField,
+} from '@island.is/application/core'
+import { m } from '../../lib/messages'
+import format from 'date-fns/format'
+import { isEstateRegistrant } from '../../lib/utils/isEstateRegistrant'
+import { format as formatKennitala } from 'kennitala'
+export const deceasedInfoFields = [
+  buildKeyValueField({
+    label: m.nameOfTheDeceased,
+    value: ({
+      externalData: {
+        syslumennOnEntry: { data },
+      },
+    }) => (isEstateRegistrant(data) ? data.estate.nameOfDeceased : ''),
+    width: 'half',
+  }),
+  buildKeyValueField({
+    label: m.nationalId,
+    value: ({
+      externalData: {
+        syslumennOnEntry: { data },
+      },
+    }) =>
+      isEstateRegistrant(data)
+        ? formatKennitala(data.estate.nationalIdOfDeceased)
+        : '',
+    width: 'half',
+  }),
+  buildDescriptionField({
+    id: 'spaceDIF',
+    space: 'gutter',
+    title: '',
+  }),
+  buildKeyValueField({
+    label: m.address,
+    value: 'La la Land 123', // TODO: address this with API about getting lögheimili
```

Comment:
I think there is a field that addresses this from the Syslumenn response in the estate info. I'll look into it

Reply:
Fixed type assertion from Registrant -> Info.
Added a default value from initial data